### PR TITLE
[Python] Add recipe for pyts package

### PR DESCRIPTION
**Pull request by johannfaouzi**
Hi,

First I would like to thank you because `conda-forge` is extremely helpful.

I would like to add my package `pyts`, which is a **Python package** for time series classification.

Relevant team: @conda-forge/help-python

Checklist

- [x] License file is packaged (see [here](https://github.com/johannfaouzi/staged-recipes/blob/ef1f71e69918e430819e271c62e73bec2d657053/recipes/pyts/meta.yaml#L58-L64) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

Best regards,
Johann

### CODE REVIEW

File: recipes/pyts/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (chrisburr) [inline recipes/pyts/meta.yaml]: Can you remove any comments that aren't recipe specific

File: recipes/pyts/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
+    - {{ compiler('c') }}
```
Comment (chrisburr) [inline recipes/pyts/meta.yaml]: Do you need a complier? At a glance this looks like a pure Python package

File: recipes/pyts/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  imports:
+    - pyts
+    - pyts.tests
+
+about:
+  home: http://github.com/johannfaouzi/pyts
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGPL.
+  # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
+  # See https://opensource.org/licenses/alphabetical
+  license: BSD 3-Clause
```
Comment (chrisburr) [inline recipes/pyts/meta.yaml]:
```suggestion
  license: BSD-3-Clause
```

File: recipes/pyts/meta.yaml (hunk 2 above)
Comment (johannfaouzi) [reply recipes/pyts/meta.yaml]: Indeed it's a pure Python package.

**Comment by johannfaouzi** [comment]
Thank you for the review. I update the file taking into account your remarks.

**Pull request merged**